### DISCUSSION TITLE
Split long guides for translating convenience

### DISCUSSION THREAD

**Discussion by CloneWith** [kylecorry31/Trail-Sense#2395] [New Feature Ideas]
Some of the guides (e.g. Photo Maps, Astronomy) are too long and have a lot of lines, which make them hard to translate and check on Hosted Weblate. Additionally this also often leads to potential `new line count` check failures.

It would be better if these guide files could be split into separate strings by paragraph or headings. And we can use Actions to automatically merge them back into one file and make them ready for application use.

**Comment by kylecorry31** [comment]
Hi, I will consider splitting these, though it may become a bit unmanageable since it doesn't support nested directories. But I would like to point out that partial translations are acceptable.

The new line error normally occurs when I update the guides. I'll see if I can make that smarter.
